### DISCUSSION TITLE
When I start up my foundry it forces me to ~/license

### DISCUSSION THREAD

**Discussion by avorial** [felddy/foundryvtt-docker#527] [Q&A] [answered]
This window is blank other then having the foundry logo. (its almost like it's waiting for me to put in something but there is no window appearing)

I get an error in the logs that says this :

FoundryVTT | 2022-11-28 23:04:57 | [warn] Software license requires signature.

I have tried to put the software licence in manually by using "FOUNDRY_LICENSE_KEY" and then pasting in my key from the foundry website, that did not work.

**Comment by avorial** [comment, marked as answer]
Solved it was weird port issues.

**Comment by MattAMonroe** [comment]
For anyone finding this and wondering how to resolve it themselves...

I was having this same issue and configuring my nginx proxy to allow WebSocket connections is what finally fixed it. Check the Nginx config page at: https://foundryvtt.com/article/nginx/

**Reply by TomCoursow** [reply]
> For anyone finding this and wondering how to resolve it themselves...
> 
> I was having this same issue and configuring my nginx proxy to allow WebSocket connections is what finally fixed it. Check the Nginx config page at: https://foundryvtt.com/article/nginx/

Thank you.
I was missing the Port as well as avorial stated but that alone didn't solve it for me...
I am using "Nginx-Proxy-Manager" WebUI and I had to enable the Checkbox for "Enable WebSocket Support" for the Proxy Host configuration which I only saw because of your comment so thanks :)
